torchx/lib/torchx/backend.ex

```diff
@@ -870,6 +870,36 @@ defmodule Torchx.Backend do
     |> to_nx(out)
   end
 
+  @impl true
+  def conv(%T{type: type} = out, t, k, opts) do
+    padding = opts[:padding]
+    strides = opts[:strides]
+    kernel_dilation = opts[:kernel_dilation]
+    feature_groups = opts[:feature_group_size]
+
+    # Not supported yet
+    # input_dilation = opts[:input_dilation]
+    # batch_groups = opts[:batch_group_size]
+    # input_permutation = opts[:input_permutation]
+    # kernel_permutation = opts[:kernel_permutation]
+    # output_permutation = opts[:output_permutation]
+
+    pad_config =
+      padding
+      |> Enum.map(fn {a, b} -> [a, b] end)
```

Comment:
shall I use Tuple.to_list here? which one is better?

Reply:
Both are fine. Given this case you always have two elements, I prefer the current version!